databricks/koalas/groupby.py

```diff
@@ -2444,14 +2451,9 @@ def __init__(
         column_labels_to_exlcude: Set[Tuple],
         agg_columns: List[Tuple] = None,
     ):
-        self._kdf = kdf
-        self._groupkeys = by
-        self._as_index = as_index
-        self._dropna = dropna
-        self._column_labels_to_exlcude = column_labels_to_exlcude
 
-        self._agg_columns_selected = agg_columns is not None
-        if self._agg_columns_selected:
+        _agg_columns_selected = agg_columns is not None
```

Comment:
nit: remove the leading underscore? `agg_columns_selected`

Reply:
Sounds good! Modified.